test/e2e/components/vl-modal.js

```diff
@@ -39,6 +39,27 @@ class VlModal extends VlElement {
         return await this._getContent();
     }
 
+    async isInViewport() {
+    	const element = await this._getDialog();
+    	const outOfViewport = await this._elementOutOfViewport(element);
+    	return outOfViewport.any == false;
```

Comment:
Is het niet beter dat _elementOutOfViewport een boolean terug geeft? Dan kan je onderstaande code gebruiken.
```JavaScript
return !(await this._elementOutOfViewport(element));
```